### DISCUSSION TITLE
can make two primary at type b

### DISCUSSION THREAD

**Discussion by fatmaAliGamal** [autobase-tech/autobase#427] [General]
how can make more than primary at one cluster

**Comment by vitabaks** [comment]
For example, create a distributed Citus cluster (TODO)

https://github.com/vitabaks/postgresql_cluster/issues/258

**Comment by fatmaAliGamal** [comment]
can me change after creating cluster one replica to  master to make two master ??

**Reply by vitabaks** [reply]
when I add citus support, I think you will have the opportunity to convert a regular cluster into a cluster with sharding based on the citus extension. 

But so far this is inaccurate since I have not yet dealt with this task.

**Comment by fatmaAliGamal** [comment]
how can handle high traffic for write data with type b?

**Reply by vitabaks** [reply]
You can contact Slack for help on this issue if you have a [subscription](https://github.com/sponsors/vitabaks) that includes the "PostgreSQL Expert Consulting" option.
